### PR TITLE
Fix SELinux 'restorecon' timed out and audit2allow failure

### PR DESCRIPTION
**Pull request by lilyeyes**
Fix SELinux 'restorecon' timed out and audit2allow failure and update the policy package

poo#107140 - [sle][security][sle15sp4] test fails in enforcing_mode_setup: command 'restorecon -R /' timed out
poo#107068 - [Tumbleweed][security] test fails in audit2allow: output not validating at "audit2allow -R -i $audit_log_short"
poo#107137 - [sle][security][sle15sp4] test fails in audit2allow: command 'sepolgen-ifgen' failed

- Related ticket:
  https://progress.opensuse.org/issues/107140
  https://progress.opensuse.org/issues/107068
  https://progress.opensuse.org/issues/107137
- Needles: NA
- Verification run:
  SLE15SP4 (4 arches):
    https://openqa.suse.de/tests/8227225
    https://openqa.suse.de/tests/8227227
    https://openqa.suse.de/tests/8227352
    https://openqa.suse.de/tests/8227233
  TW: https://openqa.opensuse.org/tests/2208921

### CODE REVIEW

**Review by tinawang123 — APPROVED** [review]
LGTM

File: tests/security/selinux/enforcing_mode_setup.pm [hunk 1]
```diff
@@ -23,8 +23,8 @@ sub run {
 
     # label system
     assert_script_run("semanage boolean --modify --on selinuxuser_execmod");
-    script_run("restorecon -R /", 600);
-    script_run("restorecon -R /*", 600);
+    script_run("restorecon -R /", timeout => 1800, die_on_timeout => 0);
```
Comment (rfan1) [inline tests/security/selinux/enforcing_mode_setup.pm:26]: Lili,

Can I know why we use 1800s now? it is 3 times as we did before.

File: tests/security/selinux/selinux_setup.pm [hunk 2]
```diff
@@ -53,10 +53,10 @@ sub run {
     # Install policy packages separately due to this bug: bsc#1177675
     if (is_sle('>=15')) {
         my @files
-          = ('selinux-policy-20200219-3.6.noarch.rpm', 'selinux-policy-minimum-20200219-3.6.noarch.rpm', 'selinux-policy-devel-20200219-3.20.noarch.rpm');
+          = ('selinux-policy-20220124-150400.121.1.noarch.rpm', 'selinux-policy-minimum-20220124-150400.121.1.noarch.rpm', 'selinux-policy-devel-20220124-150400.121.1.noarch.rpm');
         foreach my $file (@files) {
             assert_script_run 'wget --quiet ' . data_url("selinux/$file");
-            assert_script_run("rpm -ivh --nosignature --nodeps --noplugins $file");
+            assert_script_run("rpm -ivhU --nosignature --nodeps --noplugins $file");
```
Comment (rfan1) [inline tests/security/selinux/selinux_setup.pm:59]: rpm -Uvh shoud be enough, IMO

**Review by rfan1 — APPROVED** [review]
lgtm

File: tests/security/selinux/audit2allow.pm [hunk 3]
```diff
@@ -79,14 +79,24 @@ sub run {
...
+            .*allow.*/sx
+        }, 600);
     validate_script_output(
         "audit2allow -R -i $audit_log_short",
         sub {
             m/
             .*require\ \{.*
-            .*type\ .*;.*
+            .*type\ init_t;.*
             #=============.*==============.*
```
Comment (jouyingbin) [inline tests/security/selinux/audit2allow.pm:98]: It is better to set as 

#============= init_t ==============.*

**Pull request merged**